argo_entrypoint.sh

```diff
@@ -43,7 +43,7 @@ done; \
 echo "DEBUG: Gitea API available";
 
 # Overwrite content before import
-cp -rv "/shared/"* "${CF_IMPORT_CONTENT}/" && echo "INFO: overwriting /content success" || echo "ERROR: overwriting /content failed"
+cp -rv "/shared/"* "${CF_IMPORT_CONTENT}/" && echo "INFO: overwriting /content success" || echo "INFO: No /content found"
```

Comment:
I am not clear on what the "No content found" message is communicating. Is it saying that the source files for the copy do not exist? Or is it saying that the target directory for the copy does not exist? Either way, this code isn't making sure that the copy is failing for those reasons -- it's just assuming that a copy failure has to be because of that.

I suggest instead adding a test up front to validate that the source and/or target exist, and only attempt the copy if they do. Otherwise print an information message about what is not found.

As it is, this change could mask actual copy failures.

Reply:
As per your suggestion we have added the validation part to check any files or directories existence in source directory
and then attempting to copy/overwrite the target directory